package.json

```diff
@@ -82,7 +82,7 @@
     "pretty-quick": "^3.1.1",
     "ramda": "^0.27.1",
     "react-i18next": "^11.3.1",
-    "rxjs": "^7.4.0",
-    "typescript": "^4.4.3"
+    "rxjs": "^6.6.7",
```

Comment:
Intentionally downgrading?

Reply:
Yeah, `@openmrs/esm-framework` depends on 6.x. Having 7.x in there introduced a type conflict.